explorer/app/components/Layout/components/Header/header.tsx

```diff
@@ -35,70 +36,76 @@ export const Header = ({ header }: Props): JSX.Element => {
     socials,
   } = header;
   const [drawerOpen, setDrawerOpen] = useState(false);
-  const desktop = useBreakpointHelper(
+  const smDesktop = useBreakpointHelper(
     BREAKPOINT_FN_NAME.UP,
-    BREAKPOINT.DESKTOP
+    BREAKPOINT.DESKTOP_SM
+  );
+  const onlySmDesktop = useBreakpointHelper(
+    BREAKPOINT_FN_NAME.ONLY,
+    BREAKPOINT.DESKTOP_SM
   );
 
-  // Set drawer open state to false on change of media breakpoint from mobile to desktop.
+  // Set drawer open state to false on change of media breakpoint from mobile to "small desktop".
   useEffect(() => {
-    if (desktop) {
+    if (smDesktop) {
       setDrawerOpen(false);
     }
-  }, [desktop]);
+  }, [smDesktop]);
 
   return (
     <AppBar elevation={1} position="fixed">
       <Toolbar sx={{ gap: 4, height: HEADER_HEIGHT }} variant="dense">
         {/* Logo */}
         <Logo logo={logo} />
         <Content
-          desktop={desktop}
+          desktopSm={smDesktop}
           drawerOpen={drawerOpen}
           onDrawerClose={(): void => setDrawerOpen(false)}
         >
           {/* Slogan divider */}
-          {slogan && desktop && (
+          {slogan && smDesktop && (
             <Divider orientation="vertical" sx={{ maxHeight: 32 }} />
           )}
           {/* Slogan */}
           {slogan && (
             <Typography
               component="div"
               sx={
-                desktop
+                smDesktop
                   ? { fontSize: 12, lineHeight: "18px", maxWidth: 180 }
                   : { px: 6, py: 2 }
               }
-              variant={desktop ? undefined : "text-body-400"}
+              variant={smDesktop ? undefined : "text-body-400"}
             >
               {slogan}
             </Typography>
           )}
           {/* Nav links */}
           <NavLinks
             center={navAlignment === ELEMENT_ALIGNMENT.CENTER}
-            links={navLinks}
+            links={getHeaderNavigationLinks(navLinks, socials, onlySmDesktop)}
           />
           {/* Socials */}
-          <Socials
-            buttonSize={desktop ? "small" : "xlarge"}
-            socials={socials}
-            sx={{
-              gap: desktop ? 2 : 4,
-              px: desktop ? undefined : 4,
-              py: desktop ? undefined : 2,
-            }}
-          />
+          {!onlySmDesktop && (
```

Comment:
We render the usual social icons for mobile, tablet, and desktop and hide them for small desktop when they will be part of the "Follow Us" menu.